packages/core/README.md

```diff
@@ -207,3 +207,52 @@ var pid = core.getState("pidToKill");
 
 process.kill(pid);
 ```
+
+#### OIDC Token
+
+You can use this library to interact with the GitHub OIDC provider and get a JWT ID token which would help to get access token from third party cloud providers.
+
+**Method Name**: getIDToken()
+
+**Inputs**
+
+audience : optional
+
+**Outputs**
+
+A [JWT](https://jwt.io/) ID Token
+
+In action's `main.ts`:
+```js
+const core = require('@actions/core');
+async function getIDTokenAction(): Promise<void> {
+  
+   let aud = ''
+   const audience = core.getInput('audience', {required: false})
+   if (audience !== undefined) 
+      aud = `${audience}`
+   const id_token = await core.getIDToken(aud)
+   const val = `ID token is ${id_token}`
+   core.setSecret(id_token)
```

Comment:
Is there any reason we should expect the caller/action developer to `setSecret`. Can't we just do it in the function?

Reply:
moved it to function